MapleServer2/Types/Player.cs

```diff
@@ -382,5 +382,10 @@ private PlayerStat AddStatRegen(PlayerStatId statIndex, PlayerStatId regenStatIn
             int postRegen = Math.Clamp(stat.Current + regen, 0, stat.Max);
             return new PlayerStat(stat.Max, stat.Min, postRegen);
         }
+
+        public static bool IsVip(Player player)
+        {
+            return player.VIPExpiration > DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+        }
```

Comment:
I would prefer something like this.
```suggestion
        public bool IsVip()
        {
            return VIPExpiration > DateTimeOffset.UtcNow.ToUnixTimeSeconds();
        }
```
And then you change in PremiumClubHandler.cs to `if (!session.Player.IsVip())`